lib/tasks/kill_postgres_connections.rake

```diff
@@ -0,0 +1,14 @@
+task :kill_postgres_connections => :environment do
+  db_name = "#{File.basename(Rails.root)}_#{Rails.env}"
+  sh = <<EOF
+ps xa \
+  | grep postgres: \
+  | grep #{db_name} \
+  | grep -v grep \
+  | awk '{print $1}' \
+  | xargs kill
+EOF
+  puts `#{sh}`
+end
+
+task "db:drop" => :kill_postgres_connections
```

Comment:
What does this file do? And what do we need it for?

Reply:
The active jobs are running in the background, and using the database, so if we want to drop it we need to kill them.
This script does that